docs/core_docs/docs/troubleshooting/errors/UNREACHABLE_NODE.mdx

```diff
@@ -0,0 +1,54 @@
+# UNREACHABLE_NODE
+
+LangGraph cannot identify an incoming edge to one of your nodes. Check to ensure you have added sufficient edges when constructing your graph.
+
+Alternatively, if you are returning [`Command`](/docs/how_to/command/) instances from your nodes to make your graphs edgeless, you will need to add an additional `ends` parameter when calling `addNode` to help LangGraph determine the destinations for your node.
```

Comment:
I think you probably meant to link to the LangGraph JS docs here? This link isn't valid in this repository.